### ISSUE TITLE
Fatal error on PHP 8.1

### ISSUE THREAD

**Issue by jrfnl** [phar-io/phive#319]
Just ran into the following error when running Phive 0.14.5 on PHP 8.1:
```
    [phive] Phive 0.14.5 - Copyright (C) 2015-2021 by Arne Blankerts, Sebastian Heuer and Contributors
    [phive] PHP Fatal error:  During inheritance of IteratorAggregate: Uncaught ArgumentCountError: Too few arguments to function PharIo\Phive\Cli\Runner::errorHandler(), 4 passed in phar:///usr/local/bin/phive/src/shared/phar/ReleaseCollection.php on line 4 and exactly 5 expected in phar:///usr/local/bin/phive/src/shared/cli/Runner.php:98
    [phive] Stack trace:
    [phive] #0 phar:///usr/local/bin/phive/src/shared/phar/ReleaseCollection.php(4): PharIo\Phive\Cli\Runner->errorHandler()
    [phive] #1 /usr/local/bin/phive(342): require('...')
    [phive] #2 phar:///usr/local/bin/phive/src/shared/repository/LocalRepository.php(16): PharIo\Phive\{closure}()
    [phive] Result: 255
    [phive] #3 phar:///usr/local/bin/phive/src/services/resolver/LocalAliasResolver.php(22): PharIo\Phive\LocalRepository->getReleasesByRequestedPhar()
    [phive] #4 phar:///usr/local/bin/phive/src/services/resolver/RequestedPharResolverService.php(28): PharIo\Phive\LocalAliasResolver->resolve()
    [phive] #5 phar:///usr/local/bin/phive/src/commands/install/InstallCommand.php(53): PharIo\Phive\RequestedPharResolverService->resolve()
    [phive] #6 phar:///usr/local/bin/phive/src/commands/install/InstallCommand.php(42): PharIo\Phive\InstallCommand->resolveToRelease()
    [phive] #7 phar:///usr/local/bin/phive/src/commands/install/InstallCommand.php(37): PharIo\Phive\InstallCommand->installRequestedPhar()
    [phive] #8 phar:///usr/local/bin/phive/src/shared/cli/Runner.php(215): PharIo\Phive\InstallCommand->execute()
    [phive] #9 phar:///usr/local/bin/phive/src/shared/cli/Runner.php(69): PharIo\Phive\Cli\Runner->execute()
    [phive] #10 /usr/local/bin/phive(349): PharIo\Phive\Cli\Runner->run()
    [phive] #11 {main} in phar:///usr/local/bin/phive/src/shared/phar/ReleaseCollection.php on line 4
```

Looks like the argumentCount issue was fixed in commit ce96f4c865487c1e84f167813d18e64fe68f8436, though I expect there may still be an underlying PHP 8.1 issue which triggered the error handler to be called in the first place.

**Comment by jaapio** [comment]
You are ahead of me :-D  I encountered the same issue useing php 8.1

**Comment by jrfnl** [comment]
@jaapio Yup, this was one of the rabbit hole domino's ;-)

**Comment by theseer** [comment]
This should be fixed with 0.15.0.
